ingestion_server/ingestion_server/cleanup.py

```diff
@@ -0,0 +1,101 @@
+import logging as log
+import time
+from psycopg2.extras import DictCursor
+from urllib.parse import urlparse
+
+
+def _cleanup_url(url):
+    """
+    Add protocols to the URI if they are missing, else return none.
+    :param url:
+    :return:
+    """
+    parsed = urlparse(url)
+    if parsed.scheme == '':
+        return 'https://' + url
+    else:
+        return None
+
+
+# Define which tables, providers, and fields require cleanup. Map the field
+# to a cleanup function that returns either a cleaned version of the field
+# or 'None' to signal that no update is required.
```

Comment:
Once this inevitably gets hairy and has lots of fields/providers included, I'm going to migrate the configuration to a YAML file